terraform/modules/happy-stack-eks/versions.tf

```diff
@@ -12,6 +12,15 @@ terraform {
       source  = "datadog/datadog"
       version = ">= 3.20.0"
     }
+
+    validation = {
+      source  = "tlkamp/validation"
+      version = "1.0.0"
+    }
   }
   required_version = ">= 1.3"
 }
+
+provider "datadog" {
```

Comment:
I'm not sure this is needed, the called or this module needs to configure their datadog provider and we have required a datadog provider for this module. The configuration should be set by the caller and not in this module.